keras/preprocessing/image.py

```diff
@@ -25,6 +25,22 @@
     pil_image = None
 
 
+if pil_image is not None:
+    _PIL_INTERPOLATION_METHODS = {
+        'nearest': pil_image.NEAREST,
+        'bilinear': pil_image.BILINEAR,
+        'bicubic': pil_image.BICUBIC,
+    }
+    # These methods were only introduced in version 3.4.0 (2016).
+    if hasattr(PIL.Image, "HAMMING"):
```

Comment:
Please use consistent quote chars. If you know at which PIL version these attributes were introduce, prefer doing a version number check instead of repeated `hasattr` calls.

Reply:
Due to the various forks of PIL and hence different versioning schemes, I'd say it is safer to use this explicit test in this case.